tfs/src/main/java/hudson/plugins/tfs/model/TeamResultType.java

```diff
@@ -0,0 +1,36 @@
+package hudson.plugins.tfs.model;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public enum TeamResultType {
+    JUNIT("junit", "JUnit", Collections.singletonList("**/TEST-*.xml")),
+    MAVEN("junit", "Maven", Arrays.asList(
+            "**/surefire-reports/TEST-*.xml",
+            "**/failsafe-reports/TEST-*.xml"
+    )),

```

Comment:
I just put these two as placeholders and I am hoping this pull request will serve as a conversation starter to decide what result types we'll want to add here.